cpp/daal/src/sycl/fill_buffer_helper.cpp

```diff
@@ -0,0 +1,82 @@
+/* file: fill_buffer_helper.cpp */
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "src/sycl/cl_kernels/fill_buffer_helper.cl"
+#include "sycl/internal/types_utils.h"
+#include "sycl/internal/execution_context.h"
+#include "sycl/internal/utils.h"
+#include "sycl/internal/types.h"
+
+namespace daal
+{
+namespace oneapi
+{
+namespace internal
+{
+namespace interface1
```

Comment:
Why interaface instead library? Need to delete

Reply:
fixed